ServerCore/Pages/Teams/Hints.cshtml.cs

```diff
@@ -147,7 +147,7 @@ private async Task PopulateUI(int puzzleId, int teamId)
                 where Puzzle.ID == puzzleId
                 select Puzzle).FirstOrDefaultAsync();
 
-            PuzzleName = puzzle?.Name;
+            PuzzleName = puzzle?.PlaintextName;
```

Comment:
Since this is player-facing, should it be html?

Reply:
Honestly terrified of breaking your event, being ultra safe.